js/app.js

```diff
@@ -30,6 +30,26 @@ var showQuestion = function(question) {
 
 	return result;
 };
+// Passes in data from API and inserts it into DOM accordingly VIa .text()
+var showAnswer = function(answer){
+	// clone our result template code. Clones the code provided us. Like a copy of the code.
+	var result = $('.templates .answerToQuery').clone();
+	//Finds the display name class and adds text from returned Data from API.
+	var displayName = result.find(".display-Name");
+	displayName.text(answer.user.display_name);
+	// Finds userprofile in template. adds in attr
+	var userProfile = result.find(".user-Profile");
```

Comment:
Watch the quote's. You started off using single quotes, then switched to double. They should be consistent.